### PR TITLE
Sandbox

### PR DESCRIPTION
**Pull request by jdsipes**

### CODE REVIEW

File: bin/deployables [hunk 1]
```diff
@@ -570,27 +570,64 @@ function copy_lambda_to_s3 () {
...
           s3 cp \
           ${DEPLOY_LAMBDA_ZIP_FULLPATH} \
-          s3://${DEPLOY_LAMBDA_S3_BUCKET}/${DEPLOY_LAMBDA_FUNCTION_NAME}/${DEPLOY_LAMBDA_FUNCTION_NAME}-${DEPLOY_LAMBDA_ZIP_VERSION}.zip
+          s3://${DEPLOY_LAMBDA_S3_BUCKET}/${DEPLOY_LAMBDA_FUNCTION_NAME}/${DEPLOY_LAMBDA_FUNCTION_NAME}.zip
 
       if [[ "$?" != "0" ]] ; then
-          echo "Error: Failed to copy lambda zip to S3 (${DEPLOY_LAMBDA_ZIP_VERSION})"
+          echo "Error: Failed to copy lambda zip to S3 (${DEPLOY_LAMBDA_ZIP_FULLPATH})"
           exit 1
       fi
```
Comment (jefforulez) [inline bin/deployables:593]: @jdsipes let's talk about strategies here.   the idea behind deploying the DEPLOY_SHA1 based tag is to have a version to test based on the commit, and not just based on a release or an explicit version number.   

as for having a DEPLOY_LAMBDA_ZIP_VERSION based version, the idea is that it's easily identifable artifact for past tags/releases.

what are your thoughts on why those aren't needed?

File: bin/deployables [hunk 2]
```diff
@@ -570,27 +570,64 @@ function copy_lambda_to_s3 () {
...
+
+    # update lambda function
+    lambda_function_arn=$( \
+        aws lambda update-function-code \
+            --profile "${DEPLOY_AWS_PROFILE}" \
+            --function-name "${DEPLOY_LAMBDA_FUNCTION_NAME}" \
+            --s3-bucket "${DEPLOY_LAMBDA_S3_BUCKET}" \
+            --s3-key "${DEPLOY_LAMBDA_FUNCTION_NAME}/${DEPLOY_LAMBDA_FUNCTION_NAME}.zip" \
+            
+    )
```
Comment (jefforulez) [inline bin/deployables:626]: aws-cli is configured by deployables to output json.  you might need to put a `| $JQ foo` in here to check the return value.

File: bin/deployables (hunk 1 above)
Comment (jdsipes) [reply bin/deployables:593]: We could, but I think that we are using the S3 versioning that, I believe, is (or can be) set up to automatically archive. Maybe we can pass those types of values using the --metadata flag. One a side note, getting the github sha in circleci should be straight forward, I wasn't sure how to get the release version... maybe that we just have to manually update in the circle.yml file?

File: bin/deployables (hunk 2 above)
Comment (jdsipes) [reply bin/deployables:626]: I think I just copy pasta'ed that from the other deploy code further down in the file, I guess I just missed the $JQ part.

File: bin/deployables (hunk 1 above)
Comment (jefforulez) [reply bin/deployables:593]: but S3 versions have nothing to do with release versions, correct?   e.g. everytime you build the zip and upload it, S3 will give you a new version.    versus we want to tag something for general release and then have a copy of that zip at the ready without having to map into S3 versions.

am i missing something here?

File: bin/deployables (hunk 2 above)
Comment (jefforulez) [reply bin/deployables:626]: 👍

**Pull request closed without merging**